webcompat/templates/about.html

```diff
@@ -70,13 +71,244 @@ <h4 class="headline-3">Reaching Out</h4>
         <h3 class="headline-1 highlight">What else do we do?</h3>
         <p>
           Besides of taking care of reported web compatibility bugs, we create tools to make things easier and fix the web.
-          Starting with <a href="./" title="webcompat.com">webcompat.com</a> over our <a href="https://addons.mozilla.org/en-US/firefox/addon/webcompatcom-reporter/" title="web compat addon for reporting bugs">web compat quick reporter</a> to in browser addons like GoFaster and other. There are ways for you to <a href="./contributors" title="help and contribute">help and contribute</a>, if you are interested. Just let us know!
+          Starting with <a href="./" title="webcompat.com">webcompat.com</a> over our <a href="https://addons.mozilla.org/en-US/firefox/addon/webcompatcom-reporter/" title="web compat addon for reporting bugs">web compatibility reporter add-on</a> and other. There are ways for you to <a href="./contributors" title="help and contribute">help and contribute</a>, if you are interested. Just let us know!
         </p>
 
         <p>
           The web is big and keeps getting bigger. To make sure it’s all compatible we need help from people in all regions of the world. The great thing is anyone can help. You don’t need to be an expert and as little as 5 minutes can make a huge difference!
         </p>
       </div>
     </article>
+  </section>  
+  <section class="grid">
+    <div class="grid-row">
+      <div class="grid-cell x3"> 
+        <h3 class="headline-1 highlight">The team</h3>
+      </div>
+
+<!-- ADAM STEVENSON -->
+      <article class="grid-row bio">
+        <div class="grid-cell x1-fix">
+          <img src="https://avatars0.githubusercontent.com/u/6587558" alt="Adam Stevenson" class="bio-img">
+        </div>
+        <div class="grid-nested x1">
+          <div class="grid-cell bio-desc x1">
+            <h4 class="headline-2 highlight">Adam Stevenson</h4>
+            <h4 class="headline-3">Canada</h4>
+            <p>
+              Works on webcompat bugs and does outreach to sites / developers. Enjoys music, biking,
+              travel and sometimes photos.
+            </p>
+          </div>
+          <p class="grid-cell bio-links x1">
+            <a class="bio-icon-twitter" href="http://twitter.com/adamopenweb" title="adamopenweb on twitter">twitter@adamopenweb </a> <br>
+            <a class="bio-icon-github" href="http://github.com/adamopenweb" title="adamopenweb on github">github@adamopenweb </a>
+          </p>
+        </div>
+      </article>
+
+<!-- BEA Rizental -->
+      <article class="grid-row bio">
+        <div class="grid-cell x1-fix">
+          <img src="https://avatars3.githubusercontent.com/u/25176023" alt="Beatriz Rizental" class="bio-img">
+        </div>
+        <div class="grid-nested x1">
+          <div class="grid-cell bio-desc x1">
+            <h4 class="headline-2 highlight">Beatriz Rizental</h4>
```

Comment:
@brizental would you prefer Bea here, rather than Beatriz?

Reply:
Ahh, yes!

Reply:
@zoepage can you change it to Bea, please? Thanks!

Reply:
sure.